src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -38,30 +38,28 @@ export default class UnsupportedBrowserModal extends React.Component<{}, IBrowse
     }
 
     handleUnsupportedBrowsers(name:string, version: string, localStorage: boolean) {
```

Comment:
In the `render` function (not shown in this diff). There's is still a reference to name being `ie` which it will never be with browser package afaik